build_debian.sh

```diff
@@ -319,8 +319,6 @@ sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y in
     python3-pip             \
     python-is-python3       \
     cron                    \
-    libprotobuf23           \
-    libgrpc++               \
```

Comment:
@vdahiya12 do we need to keep libgrpc?

Reply:
@yxieca we dont need this for the new driver, But sonic-telemetry might need this. @hui-ma to comment.

Reply:
For reference, this was added as part of a commit adding support for Nokia IXR7250E and to support the Nokia NDK about 2 months ago. From what I can tell, the grpcio python package is needed (and installed in the pmon docker container), but the C/C++ libraries aren't needed there.

Reply:
I agree @mlok-nokia , could you please clarify why you install the mentioned grpc packages in host?

Reply:
The Nokia platform device manager is running the host which requires grpc and protobuf libs.

Reply:
@mlok-nokia is this just python scripts or is there a C/C++ library? If just python scripts, I don't see anything in the documentation or the quick start guide that says the C/C++ libraries also need to be installed.

Reply:
Both C/C++ and python.

Reply:
Got it, I'll revert this change.

One things is that the ++ is being treated as regex, so the development headers are being installed as well. I'm assuming the development headers aren't needed at runtime, so I'll make sure the runtime libraries are installed.

Reply:
Thanks, The headers are not needed.